internal/test/helpers.go

```diff
@@ -42,16 +43,12 @@ func CreateMntNsFilterMap(t testing.TB, mountNsIDs ...uint64) *ebpf.Map {
 		MaxEntries: 1024,
 	}
 	m, err := ebpf.NewMap(mntnsSpec)
-	if err != nil {
-		t.Fatalf("Failed to create eBPF map: %s", err)
-	}
+	require.Nil(t, err, "Failed to create eBPF map: %s", err)
 	t.Cleanup(func() { m.Close() })
 
 	for _, mountnsid := range mountNsIDs {
-		if err := m.Put(mountnsid, one); err != nil {
-			m.Close()
-			t.Fatalf("Failed to update eBPF map: %s", err)
-		}
+		err := m.Put(mountnsid, one)
+		require.Nil(t, err, "Failed to update eBPF map: %s", err)
```

Comment:
I prefer to have those in separated lines. IMO it's easier to read and also the lines are shorter. (In this case the line is not too long because the function called is rather small, `m.Put(mountnsid, one)`, however it could be much longer)